User can open finalized workout in printable format.

johnhaup opened the pull request:
![screen shot 2017-06-21 at 2 14 16 pm](https://user-images.githubusercontent.com/26287153/27407107-016404f2-568c-11e7-93c3-bb7d20882434.png)
[testprint.pdf](https://github.com/johnhaup/crunchtime/files/1092968/testprint.pdf)

victormejia commented on index.html:
```diff
@@ -49,8 +49,8 @@ <h2 class="text-center">Total Workout Time</h2>
       <div class="row">
-        <div class="col-sm-7 main-divs">
-          <div class="panel panel-default ex-panel">
+        <div class="col-sm-7" id="chooseEx">
```
`choose-ex` here for id

victormejia commented on index.html:
```diff
@@ -49,8 +49,8 @@ <h2 class="text-center">Total Workout Time</h2>
-        <div class="col-sm-7 main-divs">
-          <div class="panel panel-default ex-panel">
+        <div class="col-sm-7" id="chooseEx">
+          <div class="panel panel-default bottom-divs">
```
is there something more semantic you can name it vs. `bottom-divs`?

victormejia commented on main.js:
```diff
@@ -131,7 +131,9 @@ function addToWorkout(newEx) {
   $info.className = 'panel-body'
   createPanelBody($info, newEx)
-  createPanelFoot($info, newEx)
+  if (newEx.name !== newEx.alt && newEx.muscle !== 'Abs') {
```
expand on `alt`, not sure what that is

victormejia commented:
also the gif isn't displaying correctly

johnhaup commented:
I think I already removed it a few commits ago.  I deleted it from terminal and added changed file to my commit.  It shows red blocked and "Binary File Removed" tooltip when I look at the files for this pull request.

victormejia commented:
you will have to use `git rm`
